proptest/src/arbitrary/_alloc/collections.rs

```diff
@@ -9,7 +9,7 @@
 
 //! Arbitrary implementations for `std::collections`.
 
-//#![cfg_attr(feature="cargo-clippy", allow(implicit_hasher))]
```

Comment:
this is just dead code? can probably remove this but won't block getting the fix in.